app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def rate_service(self, wash_rating: int) -> None:
+        self.average_rating = (
+            (self.average_rating * self.count_of_ratings + wash_rating)
+            / (self.count_of_ratings + 1)
+        )
+        self.average_rating = round(self.average_rating, 1)
+        self.count_of_ratings += 1
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = car.comfort_class * (self.clean_power - car.clean_mark) * \
+            self.average_rating / self.distance_from_city_center
+        return round(price, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def serve_cars(self, cars: list) -> float:
```

Comment:
specify type annotation for `cars`

Reply:
But I already specified the annotation type for the car variable in the serv_cars function. The function accepts a list of cars "(self, cars: list)". Or did I misunderstand you?

Reply:
You have already fixed it)

Reply:
It dawned on me that I needed to clarify the type of objects in the list